plugin/plugins/git/git.go

```diff
@@ -33,43 +31,19 @@ func (*gitPlugin) Version() string {
 }
 
 func (*gitPlugin) Init(_ *plugin.Environment) error {
+	// register a custom identifier in the reserved range for import of "zlib-encoded git objects."
+	// TODO: give this a name.
+	multicodec.RegisterDecoder(uint64(mc.ReservedStart+mc.GitRaw), decodeZlibGit)
```

Comment:
I think so, because this plugin is "zlib-encoded git object" rather than just "git object", and only the git multicodec is registered in its `init`, this one gets a special codec in the reserved multicodec range.